scripts/autotracking.lua

```diff
@@ -160,15 +160,17 @@ function updateTail(segment)
     local item = Tracker:FindObjectForCode("tail")
     if item then
         local tail = ReadU8(segment, 0x602D)
-        local bahumat = ReadU8(segment, 0x620E)
+        local bahamut = ReadU8(segment, 0x620E)
         if AUTOTRACKER_ENABLE_DEBUG_LOGGING then
-            print(item.Name, ruby, bahumat, item.CurrentStage)
+            print(item.Name, tail, bahamut, item.CurrentStage)
         end
-
-        if tail > 0 then
+		
+        if tail > 0 and bahamut == 1 then
             item.CurrentStage = 1
-        elseif bahumat & 0x02 ~= 0 then
-            item.CurrentStage = 2
+		elseif bahamut == 2 and tail == 1 then
```

Comment:
This should use an 'OR' rather than two branches if it means the same thing. Also a comment would be helpful

Reply:
See above comment. Working on fix now. Next merge will address.